proxy/hdrs/MIME.h

```diff
@@ -1790,7 +1790,7 @@ MIMEHdr::get_cooked_pragma_no_cache()
   -------------------------------------------------------------------------*/
 
 inline void
-MIMEHdr::set_cooked_cc_need_revalidate_once()
+MIMEHdr::set_cooked_cc_need_revalidate_once() // NOLINT(readability-make-member-function-const)
```

Comment:
@bryancall This is a good example. The `readability-make-member-function-const` suggests make this const function, but this is changing the `m_mask`.
If we run `clang-tidy` with `readability-make-member-function-const`, we need to check all suggestions. :(